SPECS/xcp-ng-xapi-storage.spec

```diff
@@ -62,6 +62,9 @@ make
 cd build
 %make_install
 
+%post
```

Comment:
If you want this to apply to the `volume-zfsvol` subpackage, then this needs to be `%post volume-zfsvol`